File: Ejemplos/ProgramacionOrientadaObjetos/src/main/java/AnestesistaCirujano.java
public class AnestesistaCirujano extends Medico implements IAnestesista, ICirujano {

    public AnestesistaCirujano(String nombre, String numeroColegiado) {
        super(nombre, numeroColegiado);
    }

    @Override
    public void administrarAnestesia() {
        System.out.println("Soy el anestesista cirujano " + getNombre() + " administrando anestesia");
    }

    @Override
    public void eliminarAnestesia() {
        System.out.println("Soy el anestesista cirujano " + getNombre() + " eliminando anestesia");
    }

    @Override
    public void operar() {
        System.out.println("Soy el anestesista cirujano " + getNombre() + " operando");
    }

    @Override
    public void suturar() {
        System.out.println("Soy el anestesista cirujano " + getNombre() + " suturando");
    }

    @Override
    public void saludar() {
        System.out.println("Hola, soy el anestesista cirujano " + getNombre());
    }
}

File: Ejemplos/Hospital-MVC-Kotlin-Mockk/src/main/kotlin/models/AnestesistaCirujano.kt
package models

import java.util.*

class AnestesistaCirujano(
    id: UUID,
    nombre: String,
    añosExperiencia: Int,
    val quirofano: Int,
) : Anestesista(id, nombre, añosExperiencia), ICirujano {
    override fun operar() {
        println("Soy el anestesista cirujano $nombre y estoy operando")
    }

    override fun calculaBonus(): Double {
        return 75.0
    }

    override fun toString(): String {
        return "AnestesistaCirujano(id=$id, nombre='$nombre', añosExperiencia=$añosExperiencia, quirofano=$quirofano)"
    }
}